Sample Applications/EditingExaminerDemo/EditingExaminerDemo.csproj

```diff
@@ -6,7 +6,6 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace>EditingExaminerDemo</RootNamespace>
     <AssemblyName>EditingExaminerDemo</AssemblyName>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
How does this work? This project still includes `[assembly:]` properties from `AssemblyInfo.cs`:

https://github.com/microsoft/WPF-Samples/blob/79ecdf0b4dd8800d71f3d582fddfc4304b7430d9/Sample%20Applications/ExpenseIt/ExpenseItDemo/Properties/AssemblyInfo.cs#L12-L19

Reply:
I have removed the AssemblyInfo.cs files. Builds are passing for me.

Reply:
the WPFSamples build is now passing